keras_cv/models/generative/stable_diffusion/image_encoder.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, img_height=512, img_width=512, download_weights=True):
 
         if download_weights:
             image_encoder_weights_fpath = keras.utils.get_file(
-                origin="https://huggingface.co/fchollet/stable-diffusion/blob/main/vae_encoder.h5",
-                file_hash="f142c8c94c6853cd19d8bfb9c10aa762c057566f54456398beea6a70a639bf48",
+                origin="https://huggingface.co/fchollet/stable-diffusion/resolve/main/vae_encoder.h5",
+                file_hash="c60fb220a40d090e0f86a6ab4c312d113e115c87c40ff75d11ffcf380aab7ebb",
```

Comment:
Does the model not work with the original link + hash?

Reply:
Perhaps we need to rebase?

Reply:
Rebase to master?

Reply:
Yeah this link + hash pair doesn't work for me for some reason?

Reply:
Interesting, which one works?  c60fb220a40d090e0f86a6ab4c312d113e115c87c40ff75d11ffcf380aab7ebb or f142c8c94c6853cd19d8bfb9c10aa762c057566f54456398beea6a70a639bf48?

Lets just make sure we have the correct one in there.